### PR TITLE
Updates/Additions to Recoverable Items folder articles

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2333.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/policy-and-compliance/recoverable-items-folder/recoverable-items-folder.md [hunk 1]
```diff
@@ -101,14 +101,18 @@ Soft-deleted items are moved to the Deletions subfolder of the Recoverable Items
...
 
-After the deleted item retention period expires, the item is moved to the Purges folder and is no longer visible to the user. When the Managed Folder Assistant processes the mailbox, items in the Purges subfolder are purged from the mailbox database.
+When the deleted item retention period expires, the item is completely removed from Exchange Server.
+
+_Don't permanently delete items until the database is backed up_ setting can affect this bevahior. If this setting is not enabled (default), only the deleted item retention period is considered to remove items from the Deletions subfolder. If this setting is enabled and the deleted item retention period is reached, items are not deleted until after the mailbox database on which the mailbox is located has been backed up.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/policy-and-compliance/recoverable-items-folder/recoverable-items-folder.md:106]:
```suggestion
The _Don't permanently delete items until the database is backed up_ setting can affect this behavior. If this setting is not enabled (default), only the deleted item retention period is considered to remove items from the Deletions subfolder. If this setting is enabled and the deleted item retention period is reached, items are not deleted until after the mailbox database on which the mailbox is located has been backed up.
```

File: Exchange/ExchangeServer2013/recoverable-items-folder-exchange-2013-help.md [hunk 2]
```diff
@@ -97,13 +97,17 @@ Permanently deleted items are moved to the Deletions subfolder of the Recoverabl
...
 
-After the deleted item retention period elapses, the item is moved to the Purges folder and is no longer visible to the user. When the Managed Folder Assistant processes the mailbox, items in the Purges subfolder are purged from the mailbox database and can't be recovered.
+When the deleted item retention period expires, the item is completely removed from Exchange 2013.
+
+_Don't permanently delete items until the database is backed up_ setting can affect this bevahior. If this setting is not enabled (default), only the deleted item retention period is considered to remove items from the Deletions subfolder. If this setting is enabled and the deleted item retention period is reached, items are not deleted until after the mailbox database on which the mailbox is located has been backed up.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/recoverable-items-folder-exchange-2013-help.md:102]:
```suggestion
The _Don't permanently delete items until the database is backed up_ setting can affect this behavior. If this setting is not enabled (default), only the deleted item retention period is considered to remove items from the Deletions subfolder. If this setting is enabled and the deleted item retention period is reached, items are not deleted until after the mailbox database on which the mailbox is located has been backed up.
```

File: Exchange/ExchangeServer2013/recoverable-items-folder-exchange-2013-help.md [hunk 3]
```diff
@@ -150,7 +154,7 @@ In Exchange 2013, single item recovery isn't enabled by default for new mailboxe
...
 
-Placing a mailbox on In-Place Hold or Litigation Hold stops the Managed Folder Assistant from automatically purging messages from the DiscoveryHolds and Purges subfolders. Additionally, copy-on-write page protection is also enabled for the mailbox. Copy-on-write page protection creates a copy of the original item before any modifications are written to the Exchange store. After mailbox is removed from hold, the Managed Folder Assistant resumes automated purging.
+Placing a mailbox on In-Place Hold or Litigation Hold stops the Managed Folder Assistant from automatically purging messages from the DiscoveryHolds, Deletions and Purges subfolders. Additionally, copy-on-write page protection is also enabled for the mailbox. Copy-on-write page protection creates a copy of the original item before any modifications are written to the Exchange store. After mailbox is removed from hold, the Managed Folder Assistant resumes automated purging.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/recoverable-items-folder-exchange-2013-help.md:157]:
```suggestion
Placing a mailbox on In-Place Hold or Litigation Hold stops the Managed Folder Assistant from automatically purging messages from the DiscoveryHolds, Deletions, and Purges subfolders. Additionally, copy-on-write page protection is also enabled for the mailbox. Copy-on-write page protection creates a copy of the original item before any modifications are written to the Exchange store. After the mailbox is removed from hold, the Managed Folder Assistant resumes automated purging.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**